server/routes/movePrisoner/index.ts

```diff
@@ -36,15 +35,13 @@ export default function movePrisonerRoutes({
   get('/select-prisoner', prisonerSelect.view)
   post('/select-prisoner', prisonerSelect.submit)
 
-  get('/select-prisoner', movePrisoner.view)
+  get('/select-hospital', movePrisoner.view)
+  post('/select-hospital', movePrisoner.submit)
 
   get('/confirm-move', movePrisonerConfirmation.view)
   post('/confirm-move', movePrisonerConfirmation.submit)
 
   get('/prisoner-moved-to-hospital', movePrisonerCompleted.view)
 
-  get('/', movePrisoner.view)
```

Comment:
this was quite odd - the root path was used for selecting the hospital even though it was in the middle of the flow